vcert/common.py

```diff
@@ -556,12 +556,17 @@ def __init__(self, req_id=None, thumbprint=None,  reason=RevocationReasons.NoRea
 
 
 CLIENT_ID = "vcert-sdk"  # type: str
-SCOPE = "certificate:manage,revoke;configuration:manage"  # type: str
+# Certificate Management scope
+SCOPE_CM = "certificate:manage,revoke"  # type: str
+# Policy Management scope
+SCOPE_PM = 'certificate:manage;configuration:manage'  # type: str
+# Full Management scope
+SCOPE_FULL = 'certificate:manage,revoke;configuration:manage'  # type: str
```

Comment:
Nit: inconsistent use of double/single quotes

Reply:
Changed to double quotes as those are the ones commonly used for plain strings.